cmd/osv-scanner/main_test.go

```diff
@@ -503,18 +522,19 @@ func TestRun_LocalDatabases(t *testing.T) {
 			exit: 0,
 		},
 	}
+
 	for _, tt := range tests {
 		tt := tt
 		t.Run(tt.name, func(t *testing.T) {
 			t.Parallel()
 
-			testDir, cleanupTestDir := createTestDir(t)
-			defer cleanupTestDir()
-
-			old := tt.args
-
-			tt.args = []string{"", "--experimental-local-db-path", testDir}
-			tt.args = append(tt.args, old[1:]...)
+			if os.Getenv("TEST_ACCEPTANCE") == "true" {
```

Comment:
we should be using our new helper here:

```suggestion
			if testutility.AcceptanceTestOn() {
```